public/content/en/gems/ndslice.md

```diff
@@ -0,0 +1,120 @@
+# Multidimensional arrays
+
+D provides support for multidimensional arrays directly in its standard library
+Phobos. It allows fast iteration, access and manipulation of multidimensional
+data. In comparison to popular software packages in other languages (e.g. `ndarray`
+in NumPy or `Array` in Julia), it doesn't become slow for computations not
+covered by preexisting functionality in a bound C-library nor forces the user
+to escape into another language. Additionally, due to its compile-time introspection
+capabilities, D enables further optimizations for superior performance.
+
+## Slices
+
+A `Slice` is a view on a multidimensional view on usually an underlying raw,
+one-dimensional array. With the `sliced` function a multidimensional array can
+be constructed - in this example a `2x3` matrix is created:
+
+```d
+auto arr = new double[6];       // allocate one-dimensional array
+auto matrix = arr.sliced(2, 3); // construct view onto array
+```
+
+As specifying the size of the to-be-allocated array is tedious, `ndslice` provides
+a `slice` function, which will automatically allocate:
```

Comment:
As the two part process of 1) allocating an array of the right size and 2) slicing it can get a bit tedious, the `ndslice` package provides a shorthand that does the whole trick in one go: